kernel/src/table_configuration.rs

```diff
@@ -88,49 +88,66 @@ impl TableConfiguration {
             version,
         })
     }
+
```

Comment:
sorry for adding few formatting changes - the no newlines were hurting my eyes..